server/src/test/java/io/crate/planner/InsertPlannerTest.java

```diff
@@ -229,9 +230,13 @@ public void testInsertFromSubQueryGlobalAggregate() {
         assertThat(projection.allTargetColumns().get(0).column().fqn(), is("name"));
         assertThat(projection.allTargetColumns().get(1).column().fqn(), is("id"));
 
-        assertThat(projection.columnSymbols().size(), is(2));
-        assertThat(((InputColumn) projection.columnSymbols().get(0)).index(), is(0));
-        assertThat(((InputColumn) projection.columnSymbols().get(1)).index(), is(1));
+        List<Symbol> columnSymbols = InputColumns.create(
+            projection.allTargetColumns(),
+            new InputColumns.SourceSymbols(projection.allTargetColumns()));
+
+        assertThat(columnSymbols.size(), is(2));
+        assertThat(((InputColumn) columnSymbols.get(0)).index(), is(0));
+        assertThat(((InputColumn) columnSymbols.get(1)).index(), is(1));
```

Comment:
Please remove, this tests the logic of `InputColumns.SourceSymbols` and thus should not be part of this test.